app/controllers/catalog_controller.rb

```diff
@@ -280,17 +309,15 @@ class CatalogController < ApplicationController
     config.add_show_tools_partial(:sms, if: :render_sms_action?, callback: :sms_action, validator: :validate_sms_params)
 
     # Custom tools for GeoBlacklight
-    # rubocop:disable Layout/LineLength
-    config.add_show_tools_partial :web_services, if: proc { |_context, _config, options|
-      options[:document] && (Settings.WEBSERVICES_SHOWN & options[:document].references.refs.map { |r| r.type.to_s }).any?
-    }
```

Comment:
Why is this removed?

Reply:
Seems like maybe the web services partial is now added by default in _show_sidebar.html.erb in GeoBlacklight so it no longer needs to be added a show tools partial separately.  (Seems that way looking at GeoBlacklight itself)

Reply:
Yeah, we don't need to explicitly add this partial any longer